gcovr/formats/html/__init__.py

```diff
@@ -33,6 +33,278 @@
 LOGGER = logging.getLogger("gcovr")
 
 
+class HtmlHandler(BaseHandler):
```

Comment:
In this file the manual merge is wrong, we've now the copyright notice and the class twice.

Reply:
Bah, I really shouldn't have used VS Code's merge tool without understanding it better it seems! Fixing.